# Goal Definition
uint8 TYPE_GOTO=0

uint8 action_type

PoseEuler pose
bool set_orientation
---
# Result Definition

---
# Feedback
